src/examples/lock/DeviceController.cpp

```diff
@@ -35,6 +36,8 @@
 #include "WDMFeature.h"
 #include "AppTask.h"
 
+#include <inttypes.h>
```

Comment:
This looks like the correct header ordering, but just want to confirm we're running these through the Google linter to enforce stuff like that.

Reply:
Did not pay much attention about the ordering. Feel free to file an issue so we address it in the future.